src/pages/settings/Report/ReportSettingsPage.js

```diff
@@ -64,7 +64,7 @@ function ReportSettingsPage(props) {
     const shouldDisableWelcomeMessage =
         isMoneyRequestReport || ReportUtils.isArchivedRoom(report) || !ReportUtils.isChatRoom(report) || _.isEmpty(linkedWorkspace) || linkedWorkspace.role !== CONST.POLICY.ROLE.ADMIN;
 
-    const shouldDisableSettings = _.isEmpty(report) || ReportUtils.shouldDisableSettings(report) || ReportUtils.isArchivedRoom(report);
+    const shouldDisableSettings = _.isEmpty(report) || ReportUtils.isArchivedRoom(report);
```

Comment:
After enabling settings for all the reports and revealing access to the notification preference page. We had to test the offline behavior for the newly enabled reports. Coming from https://github.com/Expensify/App/issues/29334 this caused a regression where the creation of a new task missed the optimistic prop `notificationPreference`.